hedwig/backends/aws.py

```diff
@@ -122,6 +122,7 @@ def __init__(self, dlq=False):
         self._sqs_resource = None
         self._sqs_client = None
         self.queue_name = f'HEDWIG-{settings.HEDWIG_QUEUE}{"-DLQ" if dlq else ""}'
+        self._error_count = 0
```

Comment:
should this be a property on `HedwigConsumerBaseBackend` so we can call `super().__init__()` instead of adding this to the subclass? i think we'd want to have this variable defined in the base class since behavior of the base class depends on it

Reply:
Yes, I thought about it but I didn't sit entirely right to add a constructor to a base class.

Reply:
Updated.